src/app/components/content/IsaacClozeQuestion.tsx

```diff
@@ -19,13 +19,21 @@ import {v4 as uuid_v4} from "uuid";
 import {Item} from "../elements/markup/portals/InlineDropZones";
 import {buildUseKeyboardSensor} from "../../services/clozeQuestionKeyboardSensor";
 
-const augmentInlineItemWithUniqueReplacementID = (idv: ClozeItemDTO) => idv ? ({...idv, replacementId: `${idv?.id}-${uuid_v4()}`}) : undefined;
+const augmentInlineItemWithUniqueReplacementID = (idv: ClozeItemDTO | undefined) => idv ? ({...idv, replacementId: `${idv?.id}-${uuid_v4()}`}) : undefined;
```

Comment:
This may also appear elsewhere. I know checking for definedness in this way is idiomatic JS, and it's fine in this case, but it shouldn't because a value of `0` or `""`, for example, is defined but would test false-y. Using the `isDefined()` utility is better because it checks for `null` and `undefined` specifically.

```suggestion
import {isDefined} from "../../services/miscUtils";

const augmentInlineItemWithUniqueReplacementID = (idv: ClozeItemDTO | undefined) => isDefined(idv) ? ({...idv, replacementId: `${idv?.id}-${uuid_v4()}`}) : undefined;
```

Reply:
I do use that utility a fair amount, not sure why I didn't add it here!